examples/hybridexampleapp/android/app/src/main/java/com/instabug/hybridsampleapp/HybridApplication.java

```diff
@@ -0,0 +1,20 @@
+package com.instabug.hybridsampleapp;
+
+import android.app.Application;
+import android.util.Log;
+import com.instabug.library.Instabug;
+import com.instabug.library.LogLevel;
+import com.instabug.library.invocation.InstabugInvocationEvent;
+
+public class HybridApplication extends Application {
+    @Override
+    public void onCreate() {
+        super.onCreate();
+        Log.v("Instabug-Hybrid", "starting from native");
+
+        new Instabug.Builder(this, "deb1910a7342814af4e4c9210c786f35")
```

Comment:
Let's use `RNInstabug` instead of the native `Instabug` to be more aligned with how we recommend hybrid apps to integrate with the RN SDK. Here's the [Dream11 document](https://instabug.atlassian.net/wiki/spaces/CP/pages/4574838874/Dream11+Hybrid+Setup) for reference.